ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/bnd/BndBuilder.java

```diff
@@ -127,6 +127,14 @@ private static void buildProjectJar(IProject project, IProgressMonitor monitor)
 				ProjectJar jar = new ProjectJar(project, CLASS_FILTER);
 				builder.setJar(jar);
 				builder.build();
+				BndErrorReporter errorReporter = new BndErrorReporter(project.getFile(BndProject.INSTRUCTIONS_FILE));
+				for (String err : bnd.getErrors()) {
+					errorReporter.addError(err, bnd.getLocation(err));
+				}
+				for (String warn : bnd.getWarnings()) {
+					errorReporter.addWarning(warn, bnd.getLocation(warn));
+				}
+				errorReporter.validateContent(null);
```

Comment:
Yes, but if one only performs a clean build, without auto-build the markers persist.
This might be an edge case, but should be done for completenes, just like other builders do it.

Reply:
A clean build always performs a full build afterwards, so I'm not sure what this case should cover, even if I disable autobuild and then perform a clean the problem is still present so removing it seems not very useful.

Reply:
> A clean build always performs a full build afterwards, so I'm not sure what this case should cover, even if I disable autobuild and then perform a clean the problem is still present so removing it seems not very useful.

Other PDE builders cover that case and I would actually expect no error markers after a clean and no build (doesn't mean everything ist correct).

Reply:
As said I don't see any value in this (if one want can 'delete' markers anyways) so I don't want to complicate this here more than required. If one has the requirement for this it could be added as a separate PR, but this is to fix that in case of error nothing is shown not the other way round ;-)

Reply:
> As said I don't see any value in this (if one want can 'delete' markers anyways) so I don't want to complicate this here more than required.

It is not that complicated, you can see it here, but in this case it's much simpler :)
https://github.com/eclipse-pde/eclipse.pde/blob/c71d0c154d36ffaccf2c06d747911c45bbe4a7c1/ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/builders/ManifestConsistencyChecker.java#L438-L467

> If one has the requirement for this it could be added as a separate PR, but this is to fix that in case of error nothing is shown not the other way round ;-)

Sometimes things should come together. :)
And since we know this now, it should be done right initially. 
We already enough inconsistency in PDE.

Reply:
Of course only the markers of the PDE.bnd should be delted.

Reply:
Done.